cmd/mungedocs/analytics.go

```diff
@@ -37,6 +38,10 @@ var (
 
 // This adds the analytics link to every .md file.
 func checkAnalytics(fileName string, fileBytes []byte) (output []byte, err error) {
+	if strings.HasSuffix(fileName, "types.go") {
```

Comment:
I'll make it a separate tool located in pkg/api/, because it doesn't share much code with the current link checker.